app/src/main/java/com/yeonae/chamelezone/view/review/ReviewCreateActivity.kt

```diff
@@ -71,13 +70,21 @@ class ReviewCreateActivity :
             when {
                 edt_review.text.isEmpty() -> shortToast(R.string.review_content)
                 uriList.isEmpty() -> shortToast(R.string.review_image)
-            }
-            if (!isCreated) {
-                isCreated = true
-                presenter.reviewCreate(memberNumber, placeNumber, content, uriList)
-                Handler().postDelayed({
-                    isCreated = false
-                }, 5000)
+                else -> {
+                    showLoading()
+                    if (!isCreated) {
+                        isCreated = true
+                        presenter.reviewCreate(
+                            memberNumber,
+                            placeNumber,
+                            content,
+                            uriList
+                        )
+                        Handler().postDelayed({
+                            isCreated = false
+                        }, 5000)
```

Comment:
```suggestion
                        },  2000~3000)
```
통신에 5초가 걸릴일이 있을까요? 또 통신 대기에 사용되는 이  millisecond도 따로 공용변수로 작성해서 일괄 적용하는게 깔끔한 코드가 될 것 같아요.
하지만, 임의의 시간을 부여하는게 아닌, 통신의 결과도출 시 초기화 해주는 방법이 좀 더 명확하지 않을까 합니다

Reply:
수정했습니다! 
ed4bc6a
97056b5

Reply:
Long type은 3000L 이렇게 뒤에 L붙이면 됩니다.

Reply:
정보 감사합니당 
수정했습니다!